test/action.test.js

```diff
@@ -466,3 +466,96 @@ tap.test('should forbid minor when target is patch', async () => {
   sinon.assert.notCalled(stubs.approveStub)
   sinon.assert.notCalled(stubs.mergeStub)
 })
+
+tap.test(
+  'should not allow merge with compatibility score lower than target score',
```

Comment:
Since in documentation stated "security updates may include compatibility scores", I would also add test for case when compat score is not provided